domain/src/test/java/org/oppia/android/domain/devoptions/ModifyLessonProgressControllerTest.kt

```diff
@@ -308,6 +315,65 @@ class ModifyLessonProgressControllerTest {
     }
   }
 
+  @Test
+  fun markFirstAndFractionsTopicsCompleted_TopicsAreCompleted() {
```

Comment:
```suggestion
  fun markFirstAndFractionsTopicsCompleted_all(or both)TopicsAreCompleted() {
```

Reply:
Changed to `markFirstAndFractionsTopicsCompleted_bothTopicsAreCompleted`.